tests/main/interfaces-time-control/task.yaml

```diff
@@ -5,34 +5,58 @@ details: |
     can access the /dev/rtc device node exposed by a slot on the OS
     snap properly.
 
-systems: [ubuntu-core-16-64]
+systems: [-opensuse-*,-fedora-*,-ubuntu-core-*]
 
 prepare: |
-    if [ "$TRUST_TEST_KEYS" = "false" ]; then
-        echo "This test needs test keys to be trusted"
-        exit
+    # Install a snap declaring a plug on time-control
+    snap try $TESTSLIB/snaps/test-snapd-timedate-control-consumer
```

Comment:
Why do you snap try rather than install local?

Reply:
Thanks. Changed. Do you know if the snap file is deleted when we use install_local, or I need to manually clean it? I don't see the .snap file is being deleted in the restores.